examples/visualization/plot_meg_sensors.py

```diff
@@ -41,8 +41,8 @@
 for system, raw in raws.items():
     # We don't have coil definitions for KIT refs, so exclude them
     ref_meg = False if system == 'KIT' else True
-    fig = plot_trans(raw.info, trans=None, dig=False, eeg_sensors=False,
-                     meg_sensors=True, coord_frame='meg', ref_meg=ref_meg)
+    fig = plot_alignment(raw.info, trans=None, dig=False, eeg_sensors=False,
+                         meg_sensors=True, coord_frame='meg', ref_meg=ref_meg)
```

Comment:
how about meg_sensors = False | True | 'all'

where 'all' means show also ref_meg?

I am trying to reduce the number of parameters here.